pkg/secrets/fetch_secret.go

```diff
@@ -20,7 +20,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-const payloadVersion = "1.0"
+// PayloadVersion defines the current payload version sent to a secret backend
+const PayloadVersion = "1.0"
+
+// CompatibleMajVersion defines the compatible maj version of payload
```

Comment:
We should not duplicate this value. Since it's only used in `read_secret.go` is should be compute there.